### ISSUE TITLE
Docker for Windows crashes whenever I attempt to install it, or display any GUI.

### ISSUE THREAD

**Issue by coredumperror** [docker/for-win#2906]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 0E40B734-8450-4A2E-8C03-BB04E5D0B0E7/20181119083942

### Expected behavior
Docker for Windows installer actually runs. Docker for Windows GUI actually opens.

### Actual behavior
When I attempt to install Docker for Windows from the [usual download location](https://store.docker.com/editions/community/docker-ce-desktop-windows), the installer presents the Windows UAC dialog, then swiftly crashes with "LightweightInstaller has stopped working". I get absolutely no other message.

I managed to install it successfully via command-line by using [Chocolatey](https://chocolatey.org/), but now when the program attempts to do anything GUI-related, it crashes in the same way. I can *start* it, and I can use `docker` commands in Git Bash to create images. But as soon as I attempt to view the GUI in any way (like opening Settings from the taskbar icon context menu), or attempt to run a container that puts up the "Docker wants to access drive D:\, Do you want to Share it?" notification and click the "Share It" button, I get "Docker for Windows has stopped working".

Interestingly enough, I *can* run the "Check for updates" command from the taskbar icon's context menu, and it'll successfully not find an update and Docker for Windows doesn't crash. But "About Docker" gives instant crash. 
"Diagnose and Feeback" -> crash.
"Kitematic" -> crash
"Sign In/Create Docker ID..." -> crash
"Restart" -> crash
"Quit Docker" - No crash. It shows the animated whale icon for a few seconds, and then closes cleanly.

When Docker for Windows crashes, I get this in the Windows Event Viewer:
```
Faulting application name: Docker for Windows.exe, version: 18.6.1.19507, time stamp: 0x5b83e31e
Faulting module name: d3d9.dll, version: 10.0.16299.98, time stamp: 0xbc4a6017
Exception code: 0xc0000409
Fault offset: 0x0000000000043577
Faulting process id: 0x3a58
Faulting application start time: 0x01d47fea4f418ab2
Faulting application path: C:\Program Files\Docker\Docker\Docker for Windows.exe
Faulting module path: C:\WINDOWS\SYSTEM32\d3d9.dll
Report Id: fb8ee47a-6d93-4655-a9eb-af0c42f8196e
Faulting package full name: 
Faulting package-relative application ID: 
```

When the installer crashes, I get this:
```
Faulting application name: Docker for Windows Installer.exe, version: 18.6.1.19507, time stamp: 0x5b83e424
Faulting module name: d3d9.dll, version: 10.0.16299.98, time stamp: 0xbc4a6017
Exception code: 0xc0000409
Fault offset: 0x0000000000043577
Faulting process id: 0x3f44
Faulting application start time: 0x01d47fea1bfa75d9
Faulting application path: D:\Downloads\Docker for Windows Installer.exe
Faulting module path: C:\WINDOWS\SYSTEM32\d3d9.dll
Report Id: 366180fe-07c0-435d-bd1b-5c504e4882c1
Faulting package full name: 
Faulting package-relative application ID: 
```

I don't know any other ways to attempt to retrieve information about these crashes, as I'm not familiar with Windows development (I'm a *nix web dev). 

However, while writing up this issue, I did finally notice that the erroring module for both the installer and the regular program is `d3d9.dll`. That's DirectX, which would actually explain why only GUI stuff crashes. Now if only I had any idea whatsoever on how to diagnose DirectX issues... DXDiag gave me [this](https://pastebin.com/4eFQWMcJ), but I have no idea how to interpret that text.

This is 100% reproducible, even after multiple reboots. I updated to the latest available Windows Updates, and it still happens. I manually installed Hyper-V and Containers, and it still happens.

I have no idea if this is new, as I had never previously installed Docker for Windows (I develop using Docker at work, but only on Mac). But I do suffer from the same crash behavior with every version of the installer going back to 18.03.1-ce-win64.

I did manage to run the Docker for Windows diagnostics command through the command line, which is how I got the diagnostic upload mentioned at the top of the issue.

  - Windows Version: Windows 10 Enterprise - Version 1709 - OS Build 16299.785
  - Docker for Windows Version: Every version going back to 18.03.1-ce-win64, at least.

I'm currently in the process of installing Windows 10 version 1803, which showed up only after I thought I'd purged the list of available Windows Updates. Hopefully installing that will fix this... 

EDIT: Nope. Installing 1803 didn't help.

**Comment by coredumperror** [comment]
I figured it out!! 

I did a little googling for that DirectX DLL file, and found [this post](https://answers.microsoft.com/en-us/windows/forum/games_windows_10/windows-10-directx-9-appsgames-all-fail-with/23ef2a8a-3cd4-4df3-af20-e87e10e334dc?auth=1). It inspired me to try shutting down one program after another, and attempting to open a Docker for Windows dialog after each one. Finally, after shutting down MSI Afterburner, Docker for Windows would actually open!

MSI Afterburner had caused me problems in the recent past as well, so I decided to just uninstall it.

Hopefully this post will help future sufferers of the problems I had!
